config/initializers/scrna_config.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# This hash is used to store constants used in pooling and chip loading calculations for samples in the scRNA pipeline.
+Rails.application.config.scrna_config = {
+  # Maximum volume of a sample in microlitres
+  maximum_sample_volume: 60.0,
+  # Minimum volume of a sample in microlitres
+  minimum_sample_volume: 5.0,
+  # Minimum volume required for resuspension in microlitres
+  minimum_resuspension_volume: 10.0,
+  # Conversion factor from millilitres to microlitres
+  millilitres_to_microlitres: 1_000.0,
+  # Number of cells required for each sample going into the pool
+  required_number_of_cells_per_sample_in_pool: 30_000,
+  # Factor accounting for wastage of material when transferring between labware
+  wastage_factor: 0.95238,
```

Comment:
In one of the pooling stories Y24-384 the calculation includes a wastage volume of 5ul. Should that also be a constant (maybe `dead_volume`)? or should that calculation instead use `wastage_factor`? Seems like a duplication.

Reply:
yes